From 958306c2c87146f56a1d264f9b207cb4946e95a4 Mon Sep 17 00:00:00 2001
From: Shiva Kerdel <shiva@exdev.nl>
Date: Fri, 11 Nov 2016 12:21:18 +0100
Subject: [PATCH 070/107] Staging: fsl-mc: include: mc: Kernel type 's32'
 preferred over 'int32_t'
Git-commit: e273d54380c59eabd4e4009fcf74d070878a080b
Patch-mainline: v4.10-rc1
References: bsc#1041022

Follow the kernel type preferrences of using 's32' over 'int32_t'.

Signed-off-by: Shiva Kerdel <shiva@exdev.nl>
Acked-by: Stuart Yoder <stuart.yoder@nxp.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Alexander Graf <agraf@suse.de>

---
 drivers/staging/fsl-mc/include/mc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl-mc/include/mc.h b/drivers/staging/fsl-mc/include/mc.h
index a781a36..1c46c0c 100644
--- a/drivers/staging/fsl-mc/include/mc.h
+++ b/drivers/staging/fsl-mc/include/mc.h
@@ -81,7 +81,7 @@ enum fsl_mc_pool_type {
  */
 struct fsl_mc_resource {
 	enum fsl_mc_pool_type type;
-	int32_t id;
+	s32 id;
 	void *data;
 	struct fsl_mc_resource_pool *parent_pool;
 	struct list_head node;
-- 
1.8.5.6

